Gemfile.lock

```diff
@@ -24,12 +24,12 @@ GEM
       sass (~> 3.4)
     jekyll-sitemap (1.1.1)
       jekyll (~> 3.3)
-    jekyll-toc (0.3.0.pre1)
+    jekyll-toc (0.3.0)
       nokogiri (~> 1.6)
     jekyll-watch (1.5.0)
       listen (~> 3.0, < 3.1)
     kramdown (1.14.0)
-    liquid (3.0.6)
+    liquid (4.0.0)
```

Comment:
Note that with liquid v4, we can prevent the useless whitespace by using `{%- ... -%}`.